cmd/tink-server/main.go

```diff
@@ -93,6 +95,11 @@ func (c *DaemonConfig) PopulateFromLegacyEnvVar() {
 	if basicAuthPass := os.Getenv("TINK_AUTH_PASSWORD"); basicAuthPass != "" {
 		c.HTTPBasicAuthPassword = basicAuthPass
 	}
+	if insecure, isSet := os.LookupEnv("INSECURE"); isSet {
```

Comment:
I think we should use `TINKERBELL_INSECURE` for consistency, and to avoid possibly conflicting with another application.

Reply:
Good catch. Thanks!

Reply:
Maybe that should be TINK_INSECURE instead of TINKERBELL_?